ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -117,11 +116,11 @@ class ManagementResources(processCounter: ProcessCounter,
     }
 
   def securedRoute(implicit user: LoggedUser): Route = {
-    path("adminProcessManagement" / "snapshot" / Segment / Segment) { (processName, savepointDir) =>
-      (post & processId(processName)) { processId =>
+    path("adminProcessManagement" / "snapshot" / Segment) { processName =>
+      (post & processId(processName) & parameters('savepointDir.?, 'cancelProcess.as[Boolean] ? false)) { (processId, savepointDir, cancelProcess) =>
```

Comment:
I think this should be send in body not as query params. Wht do you think?

Reply:
I left savepointDir as query param since it's an optional parameters and being a query param is it much simpler to omit it when making the request. Sending empty body is not necessary when you would like to stick to the default value.